helix-core/src/main/java/org/apache/helix/task/TaskConstants.java

```diff
@@ -49,4 +49,24 @@
   public static final String PREV_RA_NODE = "PreviousResourceAssignment";
 
   public static final boolean DEFAULT_TASK_ENABLE_COMPRESSION = false;
+  /**
+   * Name of the JAR file for the task dynamically loaded in {@link TaskStateModel}
+   */
+  public static final String TASK_JAR_FILE_KEY = "JAR_FILE";
```

Comment:
Why is this change part of this PR?

Reply:
Sorry I created the local JarLoader branch from my local DynamicTaskConfig branch. I thought the two would be the same but apparently they're not. Let me rebase and push again.